releasing_guide.md

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
This file is created in #126 as well

Reply:
I initial added to the wrong branch and forgot to remove it! Let's keep it in #126 - it is now removed from this PR.